### PR TITLE
IBX-397: Implement the new design for content item editing screen

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/IBX-397
| Related to |  https://github.com/ezsystems/ez-support-tools/pull/97
||https://github.com/ezsystems/ezplatform-workflow/pull/193
||https://github.com/ezsystems/ezplatform-segmentation/pull/28
||https://github.com/ezsystems/ezplatform-site-factory/pull/124
||https://github.com/ezsystems/ezplatform-calendar/pull/75
||https://github.com/ezsystems/ezplatform-page-builder/pull/765
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | yes
| Tests pass?   | no
| Doc needed?   | ??
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Redesign content create and main layout

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+(function(global, doc, eZ) {
+    let sectionsCords = {};
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:2]:
```suggestion
    const sectionsCoords = {};
```

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        if (isVerticalScrollVisible()) {
+            global.scrollTo({
+                top: sectionsCords[anchorTargetSection],
+                behavior: 'smooth',
+            });
+        } else {
+            showSection(anchorTargetSection);
+        }
+    };
+    const getSectionsCords = () => {
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:32]: `get...` can be misleading if a function doesn't return anything.

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        let startSection = 0;
+        let endSection = 0;
+
+        doc.querySelectorAll('.ibexa-anchor-navigation-sections__section').forEach((section) => {
+            const { anchorSectionId } = section.dataset;
+
+            startSection = endSection;
+            endSection = section.offsetHeight;
+            sectionsCords[anchorSectionId] = startSection;
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:41]: coordinates are usually paired values, not a single one.

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    doc.querySelectorAll('.ibexa-anchor-navigation-menu__btn').forEach((button) => {
+        button.addEventListener('click', navigateTo);
+    });
+
+    doc.addEventListener('scroll', (event) => {
+        const pos = global.scrollY + scrollOffset;
+
+        doc.querySelectorAll('.ibexa-anchor-navigation-sections__section').forEach((section) => {
+            const { anchorSectionId } = section.dataset;
+
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:54]:
```suggestion
```

File: src/bundle/Resources/public/js/scripts/admin.content.edit.js [hunk 5]
```diff
@@ -81,37 +80,41 @@
...
+            doc.querySelectorAll('.ibexa-anchor-navigation-menu__btn').forEach((btn) => {
+                btn.classList.remove('ibexa-anchor-navigation-menu__btn--invalid');
             });
 
-            invalidTabs.forEach((invalidInputs) => {
-                invalidInputs.forEach((invalidInputKey) => {
-                    doc.querySelector(`#item-${invalidInputKey}`).classList.add('ez-tabs__nav-item--invalid');
+            invalidSections.forEach((sections) => {
+                sections.forEach((invalidSectionId) => {
+                    doc
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.content.edit.js:98]: I'm not sure if this is the best code styling.

File: src/bundle/Resources/public/js/scripts/admin.content.edit.js [hunk 6]
```diff
@@ -81,37 +80,41 @@
...
 
             focusOnFirstError();
         }
     };
-
     const isAutosaveEnabled = () => {
         return eZ.adminUiConfig.autosave.enabled && form.querySelector('[name="ezplatform_content_forms_content_edit[autosave]"]');
     };
+    const fitContent = () => {
+        doc
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.content.edit.js:111]: as above

File: src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js [hunk 7]
```diff
@@ -38,14 +38,15 @@
                 };
                 const extraClasses = tooltipNode.dataset.extraClasses || '';
                 const placement = tooltipNode.dataset.placement || 'bottom';
-                const container = tooltipNode.dataset.tooltipContainerSelector ?
-                    tooltipNode.closest(tooltipNode.dataset.tooltipContainerSelector) :
-                    'body';
+                const container = tooltipNode.dataset.tooltipContainerSelector
+                    ? tooltipNode.closest(tooltipNode.dataset.tooltipContainerSelector)
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js:42]: I think this not our prettier?

File: src/bundle/Resources/public/scss/_icons.scss [hunk 8]
```diff
@@ -54,7 +54,38 @@
     }
 }
 
-.ez-side-menu,
+.btn-icon {
+    .ez-icon {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_icons.scss:58]: rebase?

File: src/bundle/Resources/public/scss/_knp-menu.scss [hunk 9]
```diff
@@ -0,0 +1,37 @@
+.ibexa-knp-menu {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_knp-menu.scss:1]: This should not be named `knp-menu`. The backend implementation may change and we have multiple knp menus on one page and each will look differently.

**Review by dew326 — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+    const isVerticalScrollVisible = () => {
+        return doc.body.scrollHeight > global.innerHeight;
+    };
+    const showSection = (anchorTargetSection) => {
+        doc.querySelectorAll('.ibexa-anchor-navigation-menu__btn').forEach((btn) => {
+            btn.classList.toggle('ibexa-anchor-navigation-menu__btn--active', btn.dataset.anchorTargetSection === anchorTargetSection);
+        });
+
+        doc.querySelectorAll('.ibexa-anchor-navigation-sections__section').forEach((section) => {
+            const { anchorSectionId } = section.dataset;
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:13]: What would you say for some consistency? Either destruct also in previous loop `btn.dataset`, or not do it here.

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+
+            section.classList.toggle('ibexa-anchor-navigation-sections__section--active', anchorSectionId === anchorTargetSection);
+        });
+    };
+    const navigateTo = (event) => {
+        const { anchorTargetSection } = event.currentTarget.dataset;
+
+        getSectionsCords();
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:21]: Maybe put definition of this func before?

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+            const { anchorSectionId } = section.dataset;
+
+            startSection = endSection;
+            endSection = section.offsetHeight;
+            sectionsCords[anchorSectionId] = startSection;
+        });
+    };
+
+    doc.querySelectorAll('.ibexa-anchor-navigation-menu__btn').forEach((button) => {
+        button.addEventListener('click', navigateTo);
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:46]:
```suggestion
        button.addEventListener('click', navigateTo, false);
```

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+    });
+
+    doc.addEventListener('scroll', (event) => {
+        const pos = global.scrollY + scrollOffset;
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:50]: maybe position insead of pos? No need to shorten it

File: src/bundle/Resources/views/themes/admin/content/edit/base.html.twig [hunk 14]
```diff
@@ -105,4 +141,4 @@
 
 {% block stylesheets %}
     {% include '@ezdesign/content/edit/stylesheets.html.twig' %}
-{% endblock %}
+{% endblock %}
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/content/edit/base.html.twig:144]: unnecessary change

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js [hunk 15]
```diff
@@ -0,0 +1,63 @@
+(function(global, doc, eZ) {
+    let sectionsCords = {};
+    const scrollOffset = 100;
+    const isVerticalScrollVisible = () => {
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:4]: maybe single line?

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+
+            section.classList.toggle('ibexa-anchor-navigation-sections__section--active', anchorSectionId === sectionId);
+        });
+    };
+    const navigateTo = (event) => {
+        let startSection = 0;
+        let endSection = 0;
+        const { anchorTargetSectionId } = event.currentTarget.dataset;
+
+        doc.querySelectorAll('.ibexa-anchor-navigation-sections__section').forEach((section) => {
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:28]: what would you say for sth like
```
let prevSection = 0;
doc.querySelectorAll('.ibexa-anchor-navigation-sections__section').forEach((section) => {
    const { anchorSectionId } = section.dataset;

    sectionsPosition[anchorSectionId] = prevSection;
    prevSection = section.offsetHeight;
});
```
as far as I understand you dont need both variables

File: src/bundle/Resources/public/js/scripts/admin.context.menu.js [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+                let currentWidth = moreButton.offsetWidth + BUTTON_MARGIN;
+
+                primaryButtons.forEach((button) => button.classList.remove('ibexa-btn--hidden'));
+                moreButton.classList.remove('ibexa-btn--hidden');
+
+                for (let i = 0; i < primaryButtons.length; i++) {
+                    const button = primaryButtons[i];
+                    const isLastButton = i === primaryButtons.length - 1;
+                    const allPreviousButtonVisible = hiddenPrimaryButtons.length === 0;
+                    const isButtonNarrowerThanMoreButton = button.offsetWidth < moreButton.offsetWidth + BUTTON_MARGIN; // ??
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/admin.context.menu.js:33]: comment?

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/views/themes/admin/ui/menu/content_edit_menu.html.twig [hunk 18]
```diff
@@ -0,0 +1,63 @@
+{% extends '@KnpMenu/menu.html.twig' %}
+
+{% block root %}
+
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/ui/menu/content_edit_menu.html.twig:4]:
```suggestion
```

File: src/bundle/Resources/views/themes/admin/ui/menu/content_edit_menu.html.twig [hunk 19]
```diff
@@ -0,0 +1,63 @@
...
+        <div class="ibexa-context-menu__secondary-buttons ibexa-context-menu__secondary-buttons--hidden">
+            {% for item in item.children %}
+                {{ block('item') }}
+            {% endfor %}
+        </div>
+    </div>
+{% endblock %}
+
+{% block item -%}
+
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/ui/menu/content_edit_menu.html.twig:27]:
```suggestion
```

File: src/bundle/Resources/views/themes/admin/ui/menu/content_edit_menu.html.twig [hunk 20]
```diff
@@ -0,0 +1,63 @@
+{% extends '@KnpMenu/menu.html.twig' %}
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/ui/menu/content_edit_menu.html.twig:1]: This menu will be used in different places not only in content edit. It should be named differently.

**Review by dew326 — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/js/scripts/admin.content.edit.js [hunk 21]
```diff
@@ -30,35 +31,33 @@
...
-        invalidTab = invalidFields[0].closest('.tab-pane');
+        if (invalidSection) {
+            const { anchorSectionId } = invalidSection.dataset;
+            const invalidButton = doc.querySelector(`[data-anchor-target-section="${anchorSectionId}"`);
 
-        if (invalidTab) {
-            const invalidTabLink = doc.querySelector(`a[href="#${invalidTab.id}"]`);
-
-            invalidTabLink.click();
+            invalidButton.click();
```
Comment (tischsoic) [inline src/bundle/Resources/public/js/scripts/admin.content.edit.js:57]: Do we click it to basically call `navigateTo` function from `admin.anchor.navigation.js`?

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js (hunk 16 above)
Comment (tischsoic) [reply src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:27]: I am not sure what you are proposing, @GrabowskiM. You only changed `startSection` name to `prevSection` and position of its declaration? Maybe I missed something in your code excerpt.

I like change of the variable name, but I wonder, do we even need `sectionsPosition`, @lucasOsti?
Because it is used only in 1 place AFAIK:
https://github.com/ezsystems/ezplatform-admin-ui/pull/1770/files#diff-2b5ffe8475c06a6575af7905596bab7de43658bb4bd13a9b658da19d8d5de2b1R36
`top: sectionsPosition[anchorTargetSectionId],`

My proposition in other comment.

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js [hunk 22]
```diff
@@ -0,0 +1,63 @@
...
+            const { anchorSectionId } = section.dataset;
+
+            sectionsPosition[anchorSectionId] = startSection;
+            startSection = section.offsetHeight;
+        });
+
+        if (isVerticalScrollVisible()) {
+            formContainerNode.scrollTo({
+                top: sectionsPosition[anchorTargetSectionId],
+                behavior: 'smooth',
```
Comment (tischsoic) [inline src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:37]: What do you think? Am I missing something? @lucasOsti 
```suggestion
    const navigateTo = (event) => {
        const { anchorTargetSectionId } = event.currentTarget.dataset;
        const anchorTargetSection = doc.querySelector(`.ibexa-anchor-navigation-sections__section[data-anchor-section-id=${anchorTargetSectionId}]`);
        const anchorTargetSectionPreviousSibling = anchorTargetSection.previousSibling;
        const scrollTop = anchorTargetSectionPreviousSibling?.offsetHeight ?? 0;

        if (isVerticalScrollVisible()) {
            formContainerNode.scrollTo({
                top: scrollTop,
                behavior: 'smooth',
```

File: src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js [hunk 23]
```diff
@@ -0,0 +1,63 @@
...
+
+            doc.querySelectorAll('.ibexa-anchor-navigation-sections__section').forEach((section) => {
+                const { anchorSectionId } = section.dataset;
+                const start = section.offsetTop;
+                const end = section.offsetHeight + section.offsetTop;
+
+                if (position > start && position < end) {
+                    showSection(anchorSectionId);
+                }
+            });
```
Comment (tischsoic) [inline src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js:60]: From my perspective, you basically try to find one and only one section and then call `showSection` on it, so in my opinion it would be more expressive to do it this way:
```suggestion
            const allSections = doc.querySelectorAll('.ibexa-anchor-navigation-sections__section');
            const activeSection = allSelections.find((section) => {
                const { anchorSectionId } = section.dataset;
                const start = section.offsetTop;
                const end = section.offsetHeight + section.offsetTop;

                return position > start && position < end;
            });
            const activeSectionId = activeSection.dataset.anchorSectionId
            
            showSection(activeSectionId);
```

**Review by GrabowskiM — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

File: src/bundle/Resources/public/scss/_side-menu.scss [hunk 24]
```diff
@@ -0,0 +1,59 @@
+.ibexa-side-menu {
+    flex: 0 0 calculateRem(80px);
+
+    &--expanded {
+        z-index: 1000;
+    }
+
+    .ez-icon {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_side-menu.scss:8]:
```suggestion
    .ibexa-icon {
```
?

File: src/bundle/Resources/public/scss/_side-menu.scss [hunk 25]
```diff
@@ -0,0 +1,59 @@
...
+            border-top-left-radius: $ibexa-border-radius;
+            border-top-right-radius: $ibexa-border-radius;
+        }
+
+        &:last-child {
+            border-bottom-left-radius: $ibexa-border-radius;
+            border-bottom-right-radius: $ibexa-border-radius;
+        }
+
+        .ez-icon {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_side-menu.scss:42]:
```suggestion
        .ibexa-icon {
```

File: src/bundle/Resources/public/scss/_side-menu.scss [hunk 26]
```diff
@@ -0,0 +1,59 @@
...
+            display: block;
+            width: calculateRem(24px);
+            height: calculateRem(24px);
+            margin: 0 auto calculateRem(5px);
+        }
+    }
+
+    .btn-dark {
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_side-menu.scss:50]: probably should not be styled by this class

File: src/bundle/Resources/views/themes/admin/ui/menu/context_menu.html.twig [hunk 27]
```diff
@@ -0,0 +1,61 @@
...
+
+		{% set element = element|default('a') %}
+		
+		<{{element}}{{macros.attributes(attributes)}}>
+			{{ block('label') }}
+		</{{element}}>
+{% endblock %}
+
+{% block label %}
+	<span class="ez-btn__label">{{ parent() }}</span>
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/ui/menu/context_menu.html.twig:60]:
```suggestion
	<span class="ibexa-btn__label">{{ parent() }}</span>
```
?

**Review by dew326 — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/scss/_extra-actions.scss [hunk 28]
```diff
@@ -177,3 +177,12 @@
         }
     }
 }
+
+// TO DO: remove when redesigning !!
```
Comment (tischsoic) [inline src/bundle/Resources/public/scss/_extra-actions.scss:181]: Should it stay? If so:
```suggestion
// TODO: remove when redesigning !!
```

File: src/bundle/Resources/public/js/scripts/admin.context.menu.js [hunk 29]
```diff
@@ -0,0 +1,53 @@
...
+        const menuButtons = [...adapatItemsContainer.querySelectorAll('.ibexa-context-menu__item > .ibexa-btn:not(.ibexa-btn--more)')];
+        const popupMenuElement = adapatItemsContainer.querySelector('.ibexa-popup-menu');
+        const showPopupButton = adapatItemsContainer.querySelector('.ibexa-btn--more');
+        const adaptiveItems = new eZ.core.AdaptiveItems({
+            items: [...adapatItemsContainer.querySelectorAll('.ibexa-context-menu__item:not(.ibexa-context-menu__item--more)')],
+            selectorItem: adapatItemsContainer.querySelector('.ibexa-context-menu__item--more'),
+            itemHiddenClass: 'ibexa-context-menu__item--hidden',
+            container: adapatItemsContainer,
+            getActiveItem: () => {
+                return adapatItemsContainer.querySelectorAll('.ibexa-context-menu__item')[0];
```
Comment (tischsoic) [inline src/bundle/Resources/public/js/scripts/admin.context.menu.js:14]: Maybe 🤔 
```suggestion
                return adapatItemsContainer.querySelector('.ibexa-context-menu__item');
```

File: src/bundle/Resources/public/js/scripts/admin.context.menu.js [hunk 30]
```diff
@@ -0,0 +1,53 @@
+(function(global, doc, eZ) {
+    const adapatItemsContainer = doc.querySelector('.ibexa-context-menu');
+
+    if (adapatItemsContainer) {
+        const menuButtons = [...adapatItemsContainer.querySelectorAll('.ibexa-context-menu__item > .ibexa-btn:not(.ibexa-btn--more)')];
+        const popupMenuElement = adapatItemsContainer.querySelector('.ibexa-popup-menu');
+        const showPopupButton = adapatItemsContainer.querySelector('.ibexa-btn--more');
+        const adaptiveItems = new eZ.core.AdaptiveItems({
+            items: [...adapatItemsContainer.querySelectorAll('.ibexa-context-menu__item:not(.ibexa-context-menu__item--more)')],
```
Comment (tischsoic) [inline src/bundle/Resources/public/js/scripts/admin.context.menu.js:9]: You can add a special class to the items and AdaptiveItems will find items on its own.

**Review by GrabowskiM — APPROVED** [review]

File: src/bundle/Resources/public/js/scripts/admin.context.menu.js [hunk 31]
```diff
@@ -0,0 +1,53 @@
...
+            triggerElement: showPopupButton,
+            onItemClick: (event) => {
+                const { relatedButtonId } = event.currentTarget.dataset;
+                const button = doc.getElementById(relatedButtonId);
+
+                button.click();
+            },
+            position: () => {
+                popupMenuElement.style.right = 0;
+            },
```
Comment (tischsoic) [inline src/bundle/Resources/public/js/scripts/admin.context.menu.js:33]: And proper styles in CSS. Would it work? 
```suggestion
            position: () => {},
```

File: src/bundle/Resources/public/scss/_anchor-navigation.scss [hunk 32]
```diff
@@ -0,0 +1,74 @@
...
+
+    &__btn {
+        display: inline-flex;
+        align-items: center;
+        width: 100%;
+        padding: calculateRem(11px) calculateRem(15px);
+        border: calculateRem(1px) solid transparent;
+        border-radius: $ibexa-border-radius;
+
+        &:before {
```
Comment (tischsoic) [inline src/bundle/Resources/public/scss/_anchor-navigation.scss:14]: CSS3 syntax - here and in other places
```suggestion
        &::before {
```
ref. https://developer.mozilla.org/en-US/docs/Web/CSS/::before#syntax

File: src/bundle/Resources/views/themes/admin/ui/component/popup_menu/popup_menu.html.twig [hunk 33]
```diff
@@ -1,15 +1,17 @@
...
             {{ include('@ezdesign/ui/component/popup_menu/popup_menu_item.html.twig', {
-                'label': item.label,
-                'class': item.class|default(''),
-            })|e('html_attr') }}
-        {% endfor %}
-    {% endblock items %}
+                'label': label,
+                'class': item.class|default('')
+            }) }}
+		{% endfor %}
```
Comment (tischsoic) [inline src/bundle/Resources/views/themes/admin/ui/component/popup_menu/popup_menu.html.twig:15]: Bad indentation?

File: src/bundle/Resources/views/themes/admin/ui/component/popup_menu/popup_menu_item.html.twig [hunk 34]
```diff
@@ -1,3 +1,3 @@
 <li class="ibexa-popup-menu__item {{ class|default('') }}">
-    {{ label }}
+	{{ label }}
```
Comment (tischsoic) [inline src/bundle/Resources/views/themes/admin/ui/component/popup_menu/popup_menu_item.html.twig:2]: Tab instead of 4 spaces? What about other places?

File: src/bundle/Resources/views/themes/admin/ui/menu/context_menu.html.twig [hunk 35]
```diff
@@ -0,0 +1,61 @@
+{% extends '@KnpMenu/menu.html.twig' %}
+
+{% block root %}
+	<ul class="ibexa-context-menu">
```
Comment (tischsoic) [inline src/bundle/Resources/views/themes/admin/ui/menu/context_menu.html.twig:4]: Tabs should be changed to spaces?

File: src/lib/Behat/PageElement/ContentUpdateForm.php [hunk 36]
```diff
@@ -25,9 +25,9 @@ public function __construct(BrowserContext $context)
         parent::__construct($context);
         $this->fields = [
             'formElement' => '[name=ezplatform_content_forms_content_edit],[name=ezplatform_content_forms_user_create],[name=ezplatform_content_forms_user_update]',
-            'closeButton' => '.ez-content-edit-container__close',
+            'closeButton' => '.ez-content-edit-container__close, .ibexa-content-edit-sidebar__header .ibexa-content-edit-sidebar__back',  // TO DO: Set one selector after redesign
```
Comment (tischsoic) [inline src/lib/Behat/PageElement/ContentUpdateForm.php:28]: TODO? Here and in other places.

**Review by tischsoic — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/scss/_extra-actions.scss (hunk 28 above)
Comment (lucasOsti) [reply src/bundle/Resources/public/scss/_extra-actions.scss:181]: > Should it stay? If so:

yes, because after whole redesign we have to change it

File: src/lib/Behat/PageElement/ContentUpdateForm.php (hunk 36 above)
Comment (lucasOsti) [reply src/lib/Behat/PageElement/ContentUpdateForm.php:28]: > TODO? Here and in other places.

yes, because after whole redesign we have to change it

File: src/bundle/Resources/views/themes/admin/ui/menu/context_menu.html.twig [hunk 37]
```diff
@@ -0,0 +1,61 @@
...
+		{% for item in item.children %}
+			{ block('item') }}
```
Comment (tischsoic) [inline src/bundle/Resources/views/themes/admin/ui/menu/context_menu.html.twig:6]: Missing `{`

**Review by tischsoic — APPROVED** [review]

**Review by dew326 — APPROVED** [review]
